docs/installation.md

```diff
@@ -34,12 +34,12 @@
 - Linux: `apt-get install python-protobuf`
 
 ### Get encrypt.so (Windows part writing need fine tune)
-We don't have the copyright of encrypt.so, please grab from internet and build your self.Take the risk as your own.
-Example build sequence:
-Create a new separate folder some here
+Due to copywrite on the encrypt.so we are not directly hosting it. Please find a copy elsewhere on the internet and compile it yourself. We accept no responsibility should you encounter any problems with files you download elsewhere.
```

Comment:
Dammit I put copyright first then talked myself in to thinking it was wrong! Haha